File: C/114.c
#include <stdio.h>
#include <stdlib.h>

// 114 - Último Dígito del Factorial - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop = 0;
    scanf("%d", &loop);
    int i;
    for (i = 0; i<loop; i++) {
        int input = 0;
        scanf("%d", &input);
        if (input >= 5) {
            printf("0\n");
        } else if (input == 4) {
            printf("4\n");
        } else if (input == 3) {
            printf("6\n");
        } else if (input == 2) {
            printf("2\n");
        } else if (input == 1 || input == 0) {
            printf("1\n");
        }
    }
    return 0;
}

File: Java/114.java
import java.util.Scanner;

/**
 *
 * 114 - Último Dígito del Factorial - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int casos = sc.nextInt();
        for (int i = 1; i <= casos; i++) {
            int numero = sc.nextInt();
            if (numero >= 5) {
                System.out.println("0");
            } else if (numero == 4) {
                System.out.println("4");
            } else if (numero == 3) {
                System.out.println("6");
            } else if (numero == 2) {
                System.out.println("2");
            } else if (numero == 1 || numero == 0) {
                System.out.println("1");
            }
        }
        System.exit(0);
    }

}
